src/clj/ataru/applications/automatic_eligibility.clj

```diff
@@ -202,3 +202,31 @@
                                application))]
         (update-application-hakukohde-review connection update))))
   {:transition {:id :final}})
+
+(defn- get-application-ids
+  [suoritukset]
+  (when-not (empty? suoritukset)
+    (jdbc/with-db-connection [connection {:datasource (db/get-datasource :db)}]
+      (->> (jdbc/query connection
+                       ["SELECT id
+                         FROM latest_applications
+                         WHERE person_oid = ANY(?)"
+                        (->> suoritukset
+                             (map :person-oid)
+                             to-array
+                             (.createArrayOf (:connection connection)
```

Comment:
Voisiko tässä käyttää ennemmin `WHERE person_oid in (?)`  ja `(->> suoritukset (map :person-oid) (join ","))`? 
Olisi vähän selkeempi eikä tarviis tehdä noin paljon muunnoksia tolle parametrijoukolle.